### PR TITLE
Update dependencies thanks to `pip-compile`

### PR DESCRIPTION
**Pull request by JohanLorenzo**
I reported a bug to pyup on December 5th. They're still working on the case. In the meantime, I manually updated the reqs

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,123 +1,140 @@
...
+    --hash=sha256:c3ba1e130c813191db95c431a18cb4d20a468e98af7a77e2181b68574481ad36 \
+    --hash=sha256:fd9ddf503110bf3d8b1d270e8c673aab29ccb3dd6abf29bae1f54e5116ab4a91
 funcsigs==1.0.2 \
-    --hash=sha512:4e583bb7081bb1d6c0fe5a6935ca03032d562b93ef5c11b51a72ce9e7ac12902451cc2233c7e6f70440629d88d8e6e9625965ee408100b80b0024f3a6204afda \
-    --hash=sha512:8f3d521413af262085cecbe12a41cf1ad36d42efe4c7c892aca58240e1412c9380da72ec159a499a8eaa18bf5013a5892cff58b240b49f3cf4d9428caca9cfe2 \
-    ; python_version < "3.0"
+    --hash=sha256:330cc27ccbf7f1e992e69fef78261dc7c6569012cf397db8d3de0234e6c937ca \
+    --hash=sha256:a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50 ; python_version < "3.0"
+# For an unknown reason, putting the python_version statement at the end doesn't work
+functools32==3.2.3.post2 ; python_version < "3.0" \
```
Comment (JohanLorenzo) [inline requirements-test.txt]: Does this error seem familiar to you, @Callek ? I've never seen this before. The pip bug tracker doesn't show anything

Comment (Callek) [reply requirements-test.txt]: I've never seen this either, so not sure... odd.

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 enum34==1.1.6 \
-    --hash=sha512:3671dc8c0c3cf2a3cbafd22feb85bcc66f4262722b59683429af0865642929622fd3213113703c0b811f7e3bf9cee0e30c7403f6971585150fc013ec01333873 \
-    --hash=sha512:4f916143fe35d41b28efb7958ffa7e10d430d66697f8150b98aae3797a058378fd9a4c201969c155f30f88c962f78e4e75e684889d5138cd2fe60406924363e9 \
-    --hash=sha512:51652525adc37bd1af1d81933f965dba9c508838d9f759c80ca1392991515a29c2c0263264a4e175b37a6ba11dca68c354774e448b19ef1bdba96be5474d93ec \
-    --hash=sha512:ac5eedcd9425b06f44c3e9029cfc3e93a266a5b797c1b8bd3ed81cbbe6c90a163245c15f1ab5ebedf03f885edeb61ebc6b341c61e31789072f29f2a3e1b4bece \
-    ; python_version < '3.4'
+    --hash=sha256:2d81cbbe0e73112bdfe6ef8576f2238f2ba27dd0d55752a776c41d38b7da2850 \
+    --hash=sha256:644837f692e5f550741432dd3f223bbb9852018674981b1664e5dc339387588a \
+    --hash=sha256:6bd0f6ad48ec2aa117d3d141940d484deccda84d4fcd884f5c3d93c23ecd8c79 \
+    --hash=sha256:8ad8c4783bf61ded74527bffb48ed9b54166685e4230386a9ed9b1279e2df5b1 ; python_version < "3.4"
```
Comment (Callek) [inline requirements.txt]: @JohanLorenzo do we need to care about this python_version being at the end too?

Comment (JohanLorenzo) [reply requirements.txt]: The other deps didn't complain about it. Only `functools32` did :/

**Pull request merged**